style-guide/assets/wds-company-logo-wikia-org.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="29" viewBox="0 0 128 29">
+  <style>
```

Comment:
So far we used inlined styles in svgs. Is there any reason why we extracted it to `<style>` tag?
In this case it makes svg size bigger.